src/17-infinite-discovery/data.md

```diff
@@ -1,6 +1,39 @@
 # How to generate the data
 
-## Quick way to generate the data
+## Export large artwork dataset
+
+```ruby
+# assuming you want to export ~30k recent artworks
+artworks = Artwork.published.for_sale.order(published_at: -1).only(:id, :_slugs, :genome, :automated_genome, :partner_genome, :visual_genome, :title, :dates, :attribution_class, :category, :medium, :price_listed, :price_currency, :tags, :auto_tags, :additional_information, :default_image, :additional_images, :default_image_id, :artist_ids, :colors, :artists).limit(30000)
```

Comment:
So looking back at @cavvia's [reccomendation](https://artsy.slack.com/archives/C05EEBNEF71/p1730975109639189?thread_ts=1730889747.418719&cid=C05EEBNEF71) this feels like we're taking a slightly different approach of just grabbing the first 30k for sale rather than the last month worth of for sale artworks. Is this intentional?

Reply:
Right, I don't think it's a huge difference, but the last 30 days worth would give us a closer match to both NWFY and Artwork Recommendations. NWFY actually limits to 21 days on app and 30 on email, whereas Artwork Recs rail is using works from the last 30 days. Note that we also exclude merch (our query for that is `merch_outcome = 'not merch'`).